### ISSUE TITLE
[FR] Support an adaptive auto-size in a panel view layout

### ISSUE THREAD

**Issue by ildar170975** [nathan-gs/ha-map-card#11]
Currently a user can define a size of a card by `card_size` option.
In a Panel view layout by default a card may occupy only a small part of available area:

![изображение](https://github.com/nathan-gs/ha-map-card/assets/71872483/660585a3-0b42-482e-839e-48c384c316bf)

To fit a map to a view, a user has to define a value of `card_size` dependently on a particular viewport (which can be different o different clients). 
Suggest to add a special value for `card_size` to force the card to occupy a whole available area.

**Comment by nathan-gs** [comment]
for this we need to discover the available room (or we could try with a height: 100%).

**Comment by ildar170975** [comment]
Tried to set a `height: 100%` by card-mod yesterday.
About a possibility to style a card by card-mod - will report in [Community](https://community.home-assistant.io/t/map-card-a-slightly-improved-map-card/693088) a bit later.

**Comment by ildar170975** [comment]
Here is a seemingly stable workaround:
```
type: custom:mod-card
card:
  type: custom:map-card
  x: 10
  y: 10
card_mod:
  style:
    map-card $: |
      .type-undefined {
        height: 100%;
      }
      div#map {
        height: 100% !important;
      }
    .: |
      map-card {
        height: 100% !important;
      }
      ha-card {
        width: 100%;
        height: 100%;
        display: flex;
        flex-direction: column;
      }
```
![изображение](https://github.com/nathan-gs/ha-map-card/assets/71872483/e35c4f80-b6a3-4d5b-b1af-61fb70e72f37)

**Comment by nathan-gs** [comment]
Thanks to @Manuel-Materazzo it's fixed in v0.6.0!
